components/cli/pkg/artifacts/apim.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package artifacts
```

Comment:
Any reason to have this in an `artifacts` package? Can't we use the `runtime` package?

Reply:
Yes, they can be put into runtime package. But there the k8s artifacts are taken from cellery installation directory instead of user_home/.cellery. Could it be changed to user_home/.cellery?

Reply:
removed artifacts package and created gcp package inside runtime package.